README.md

```diff
@@ -42,9 +42,22 @@ An example of the `service` sub-command
 pd service list
 ```
 
-
 ### Client Library
 
+#### NOTICE: Breaking API Changes in master branch
```

Comment:
This isn't technically true yet, but I plan to merge open PRs soon that will make this true... so may as well prematurely optimize. :smile: